src/app/BUILD.gn

```diff
@@ -140,6 +140,8 @@ static_library("interaction-model") {
     "CASEClientPool.h",
     "CASESessionManager.cpp",
     "CASESessionManager.h",
+    "CommandSender.cpp",
+    "CommandSender.h",
```

Comment:
We should probably move the Read/Write clients in here too.  It's odd to have these conceptually similar objects in different places.

Reply:
Based on header linkage I believe this is correct: clients belong here:

![image](https://github.com/project-chip/connectedhomeip/assets/1832280/d2352363-1d26-4c2b-9724-800a118777ae)

will make the change.

Reply:
Done and also pulled in `InteractionModelHelper.h` as that was very coupled to IM:

![image](https://github.com/project-chip/connectedhomeip/assets/1832280/e62ac9fe-f944-49be-b2d0-8b56ea35d8a1)